src/documents/showcase/Bring It On!.md

```diff
@@ -0,0 +1,9 @@
+```
+layout: showcase
+title: "Bring It On!"
+featured: true
+itch: https://axolstudio.itch.io/bring-it-on
+steam: https://store.steampowered.com/app/2020460/Bring_It_On/
+website: https://axolstudio.com/projects/bring_it_on/
+date: "05-10-2022"
```

Comment:
Feel free to update dates if these games get released to consoles. Once sorting by date is added